UI/Web/src/app/cards/card-detail-layout/card-detail-layout.component.scss

```diff
@@ -5,11 +5,21 @@
     height: calc((var(--vh) *100) - 157px);
     margin-bottom: 10px;
 
+    &.expanded {
+      height: calc((var(--vh) *100) - 520px);
```

Comment:
This is the initial height right? Because the top part may be a smart filter and be longer than the original height.

Reply:
It's the 157px + the height of the filter area. I inspected and just grabbed that value. If that value is dynamic then I need to figure out a way to grab that value.

Reply:
Yes it will always be dynamic as the section a) grows as more filters are added and b) can be prefiltered by a smart filter. 

You'll likely need to use document.querySelector to get the height.